api/src/repositories/submission-repository.ts

```diff
@@ -48,6 +48,10 @@ export interface ISubmissionModel {
 export interface ISubmissionModelWithStatus extends ISubmissionModel {
   submission_status: string;
 }
+
+export interface IDatasetId {
```

Comment:
Should we consider a different name for this interface?

Reply:
Namely I wouldn't expect an object of type "dataset ID" to have a property "uuid" which actually contains said ID.

Reply:
Good point, though this is going to be removed, so easy fix haha